app/src/main/java/com/demoapp/database/DataRepository.kt

```diff
@@ -82,5 +68,4 @@ class DataRepository(val dataDuo: DataDuo, val mService: RetrofitInterface) {
             }
         }
     }
-
 }
```

Comment:
end line. Change this in your Android Studio settings to ensure end line in all files.